colorama/tests/ansi_test.py

```diff
@@ -43,6 +43,21 @@ def testForeAttributes(self):
         self.assertEqual(Fore.LIGHTCYAN_EX, '\033[96m')
         self.assertEqual(Fore.LIGHTWHITE_EX, '\033[97m')
 
+        # Check RGB function
+        self.assertEqual(Fore.RGB(0,0,0), '\033[38;2;0;0;0m')
+
+        self.assertEqual(Fore.RGB(255,0,0), '\033[38;2;255;0;0m')
+        self.assertEqual(Fore.RGB(0,255,0), '\033[38;2;0;255;0m')
+        self.assertEqual(Fore.RGB(0,0,255), '\033[38;2;0;0;255m')
+
+        #test full range
+        '''
```

Comment:
I left this and the other 'full test' within comments because I'm unsure if it's a good idea or not. It's not tested but it _should_ work

Reply:
I understand the uncertainty. In my opinion, I agree the nested 'full test' loops are not helpful - because I cannot imagine a failure mode of the code which this test might catch. I thank you for presenting them here as part of this PR, but think they can be deleted before this code gets merged. (but, see top level, comment, I don't think this PR is going to get merged -sorry!)

Incidentally, a great heuristic for deciding these questions of "how many combinations should I test" is to use TDD. The core loop is to add a very simple next test, then making it pass using the simplest possible code change. Given that, it becomes very clear that the tests you *need* (in order to justify a wanted code change) are exactly the ones you end up writing. Any other tests, such as the 'full test loops' here, never get written, because adding them would not require you to make any changes to the code-under-test. So long as one properly observed the "simplest possible code" part of the earlier steps, this demonstrates these tests are not necessary, and the question of creating them never arises.